src/components/RefundButton.tsx

```diff
@@ -169,15 +168,18 @@ const RefundButton = ({
                             msg = t("locktime_not_satisfied");
                         }
                         log.error(msg);
-                        setNotification(msg);
+                        notify("error", msg);
+                        report(swap().id, msg, jsonError);
```

Comment:
I am afraid that catching errors that are being thrown here is not going to be good enough. Can we hook into the logging library and also record every log statement?